### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by caksoylar** [precondition/keymapdb#60] [Ideas]
Do you have any plans to implement dark mode?

While I think it's nice to have in general, the main reason I am asking is some of the linked SVGs created with keymap-drawer support auto light/dark mode and that doesn't look great when the OS/browser is set to dark mode (since the background is transparent):
![image](https://github.com/user-attachments/assets/572e368a-bec7-4494-a654-96a50c8605ee)

Obviously this is something that can be solved on the user side (e.g. I used an image with light mode only) but it'd be nice if users didn't have to worry about that, especially if they link to an image that is also used in e.g. Github READMEs where it's nice to have the auto mode.

IMO a good solution would be to force `img`s to light mode in the web page, ignoring the OS/browser dark mode. I am not sure how feasible that is but I am not knowledgable with web dev in general.

**Comment by precondition** [comment]
At some point, I tried to develop a dark theme for the web site but all the light-themed keymap pictures were way too bright. If the web site is mostly composed of big light-themed pictures, I figured it did not make much sense to develop a dark theme if I wasn't also willing to develop a dark version of every picture as well.

I did not know about the light-dark theme support of keymap-drawer's SVGs though. I can easily force `img` to be in light mode by adding `color-scheme: light` in the CSS so I'll do that.

**Reply by caksoylar** [reply]
I think that's a fair justification, yes. And if the CSS tweak works then that solves the keymap-drawer issue certainly. (edit: Just noticed that it's already fixed, thanks.)

I don't know how many people check it, but adding something to the effect of "pictures should work well with light theme, especially if they use transparent backgrounds" to https://github.com/precondition/keymapdb/blob/main/db_fields_reference.md#keymapimage-string might be useful.
